list2/ex33/index.js

```diff
@@ -0,0 +1,46 @@
+import { generateMatrix, printMatrix } from "../helper.js";
+
+function secondaryDiagonal({ matrix, rowCount }) {
+  let diagonalMean = 0;
+
+  for (let rowIndex = 0; rowIndex < rowCount; rowIndex++) {
+    diagonalMean += matrix[rowIndex][rowCount - 1 - rowIndex];
+  }
+  return diagonalMean / rowCount;
```

Comment:
There is an `Array.prototype` function that is usually used for such use cases. Please, try to use it instead.